src/main/perl/ObjectText.pm

```diff
@@ -0,0 +1,293 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+# ${build-info}
+
+package CAF::ObjectText;
+
+use strict;
+use warnings;
+
+use CAF::Object qw(SUCCESS);
+use parent qw(CAF::Object);
+
+use overload ('""' => '_stringify');
+
+use Readonly;
+Readonly::Scalar my $DEFAULT_USECACHE => 1;
+
+=pod
+
+=head1 NAME
+
+CAF::ObjectText - Base class for handling text
+
+=head1 SYNOPSIS
+
+Define subclass via
+    package SubClass;
+    use parent qw(CAF::ObjectText);
+
+    sub _get_text
+    {
+        my ($self) = @_;
+        return "actual text";
+    }
+
+And use it via
+    my $sc = SubClass->new(log => $self);
+    print "$sc"; # stringification
+
+    $sc = SubClass->new(log => $self);
+    # return CAF::FileWriter instance (text already added)
+    my $fh = $sc->filewriter('/some/path');
+    if (!defined($fh)) {
```

Comment:
creating/closing filewriter instance could throw an error, which is not catched in this example. but this code does not throw any errors itself, so as an example of how to use this module, this is sufficient.